pkg/build/util/generate.go

```diff
@@ -13,6 +14,7 @@ import (
 // GenerateBuildFromConfig creates a new build based on a given BuildConfig. Optionally a SourceRevision for the new
 // build can be specified.  Also optionally a list of image names to be substituted can be supplied.  Values in the BuildConfig
 // that have a substitution provided will be replaced in the resulting Build
+// You NEED to update BuildConfig object after calling this!
```

Comment:
The same "type of thing".

----- Original Message -----
> > @@ -13,6 +14,7 @@ import (
> >  // GenerateBuildFromConfig creates a new build based on a given
> >  BuildConfig. Optionally a SourceRevision for the new
> >  // build can be specified.  Also optionally a list of image names to be
> >  substituted can be supplied.  Values in the BuildConfig
> >  // that have a substitution provided will be replaced in the resulting
> >  Build
> > +// You NEED to update BuildConfig object after calling this!
> 
> @smarterclayton not sure if i misconveyed something, but the thing that sets
> LCT today is the image_change_controller (which is basically just another
> build client like start-build and the webhook).  Builds trigged via
> start-build and the webhook do not go through the image_change_controller,
> so that is not the central place to do this currently.
> 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1275/files#r26407709